geo-location.html

```diff
@@ -142,16 +142,30 @@
        *   @param {Number} detail.longitude Longitude of the current position.
        */
 
+      detached: function () {
+        if (watch_ !== null) {
```

Comment:
Sorry one last thing. Now that I think about it, we can probably just moved the null check into the functional and have calling locations just call it with checking.